init/cli/cli.go

```diff
@@ -18,29 +18,11 @@ import (
 	"github.com/palantir/pkg/cli"
 )
 
-func handleError(ctx cli.Context, err error) int {
-	switch theError := err.(type) {
-	case nil:
-		return 0 // No error
-	case *ErrorResponse:
-		ctx.Errorf(theError.Error())
-		return theError.exitCode
-	case *SuccessResponse:
-		if theError.msg != "" {
-			ctx.Printf(theError.msg)
-		}
-		return theError.exitCode
-	default:
-		return 1 // Some other, unknown error
-	}
-}
-
 func App() *cli.App {
 	app := cli.NewApp()
 	app.Name = "go-init"
-	app.Usage = "A simple init.sh - style service launcher CLI"
-	app.ErrorHandler = handleError
+	app.Usage = "A simple init.sh-style service launcher CLI."
 
-	app.Subcommands = []cli.Command{statusCommand(), startCommand()}
+	app.Subcommands = []cli.Command{startCommand(), statusCommand(), stopCommand()}
```

Comment:
I think sls-packaging supports a `console` command: https://github.com/palantir/sls-packaging/pull/29

Also, wasn't there functionality to discover git-style subcommand scripts?

Reply:
I explicitly only want to support `start`, `stop`, and `status`, since those are the only arguments that are in the spec. `search?utf8=✓&q="init.sh+console"&type=` on the end of the internal repo base URL shows that `console` isn't used by the tooling, so it seems fine to remove. We can have a conversation about individual cases for the subcommand scripts if those come up, but I'd want a really good reason to include stuff that's outside of the spec.

Reply:
I don't think it's really acceptable to break workflows that people may be relying on without warning, even if they aren't in the spec. If we're converting bash to go, then we need to maintain the [previous behaviour](https://github.com/palantir/sls-packaging/blob/68497f1505f38f40560fec1015ca4871854f559f/gradle-sls-packaging/src/main/resources/init.sh#L163):

```
        echo "Usage: $0 {status|start|stop|console|restart|check${COMMANDS}}"
```

Searching internally, it seems people are relying on `init-reload.sh`.  If we want to get rid of these commands, we could do a release of sls-packaging that deprecates them (and warns people that they shouldn't be used), then do a major release later that removes them.

Reply:
Yep, would prefer to go the route of "deprecate everything that isn't start/stop/status in sls-packaging then major version bump when we bundle this in sls-packaging instead" rather than "drop-in replacement for the current init.sh without sls-packaging major version bump"

(also obviously we will still have init.sh but it will be a simple pass-through to this binary)